### ISSUE TITLE
How to initialise yolov3 network with yolov3.weights file only till the last few layers?

### ISSUE THREAD

**Issue by BornInWater** [AlexeyAB/darknet#917]
As mentioned in the extensive and elaborate documentation, to train yolov3 on custom objects we can initialise the network with darknet53.conv.74 file and train the rest of the layers. 
However, due to low volume of data and relevancy to my task, I want to retain the weights from yolov3.weights till almost the last layer and only train the rest with my data. How should i go about doing that?

Thank you

**Comment by salbatron** [comment]
use partial function:

https://github.com/AlexeyAB/darknet/blob/1d532aa97fd280dede27c3cd0d6b81665faca284/build/darknet/x64/partial.cmd#L21

**Comment by AlexeyAB** [comment]
Or just train:
`darknet.exe detector train data/obj.data yolov3_obj.cfg yolov3.weights -clear`

**Comment by BornInWater** [comment]
@AlexeyAB , What does -clear do ?

**Comment by BornInWater** [comment]
@salbatron , I tried the command. Am i correct in assuming that the partial command parses yolov3.weights file and creates a new weight file with parameters till the specified number[like 105]?

**Comment by salbatron** [comment]
@BornInWater yes you are right. you can choose how many layers u want to save in new weight file (like 105)

**Comment by AlexeyAB** [comment]
> What does -clear do ?

It just clears iteration number, so you need not to change max_batches= for training.

**Comment by BornInWater** [comment]
So, as in the below image:
<img width="559" alt="screen shot 2018-05-30 at 3 27 54 pm" src="https://user-images.githubusercontent.com/36148827/40713669-205c788a-641e-11e8-9a65-12602068af49.png">
The detection layers at 82, 94, 106 are the only layers are dependent on number of classes and number of filters based on this equation: filters=(classes + 5)x3) say at 82 shouldnt affect the succeeding layers right? 
So, my question here is, is it possible to change the yolo-obj.cfg file for training such that it corresponds to the number of classes i have but still use weights till say layer 105?
